__tests__/app.test.js

```diff
@@ -53,22 +53,14 @@ describe('GET /api/categories', () => {
 });
 
 describe('GET /api/reviews/:review_id', () => {
-    test('200: Ensures that what is passed IS an Object', () =>{
-        return request(app)
-        .get('/api/reviews/2')
-        .expect(200)
-        .then(({body}) => {
-            const reviews = body.reviews;
-            expect(reviews).toBeInstanceOf(Object);
-        });
-    });
     test('200: Accepts the review with the given id', () => {
         return request(app)
         .get('/api/reviews/2')
         .then(({body}) => {
-            const {review}= body.reviews;
+            const review = body.review;
+            console.log(review);
             (review) => {
-                expect.objectContaining({
+                expect.toMatchObject({
```

Comment:
This test just needs a slight tweak as it isn't actually running as intended at the moment.

We should have a statement similar to `expect(review).toMatchObject({})`

Once this is running correctly, you may notice that one of your values is actually a different data type than expected 👀